app/views/posts/index.html.erb

```diff
@@ -1,3 +1,25 @@
-<h1>
-    These are the users posts
-<h1>
\ No newline at end of file
+
+<div class="posts_index_user">
+    <div class="posts_index_user_image">
+        <img src=<%=@user.photo%>>
+    </div>
+    <div class="posts_index_user_details">
+
+        <h3 class="posts_index_user_details_username"><%=@user.name%></h3>
+        <p class="posts_index_user_details_number_of_posts">Number of posts: <%=@user.posts_counter%></p>
+    </div>
+</div>
+
+<div class="posts_index_user_posts">
+    <%@posts.each do |post|%>
+    
+        <%= render partial: "post_card", locals: {post: post, comments: @comments, user: @user}%>
+
+    <%end%>
+</div>
+    
+<div class="posts_index_pagination_button">    
+    <button>
+        Pagination        
+    </button>
+</div>
```

Comment:
As pointed out by the previous reviewer, the post index page should display the posts with recent five comments. However, in your application, though it displays the comments counter correctly, it doesn't display the recent five comments. Kindly fix this issue. 
| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/216759679-40398b15-1269-4ed4-a21d-4d19dde6f0c7.png) | ![image](https://user-images.githubusercontent.com/33268927/216759721-3580907a-e833-448c-87c5-178e91d55e42.png) |

Reply:
Thank you but I think I would need more context to this as I implemented this and upon checking just now, works just fine: 

[GIF](https://www.webmobilefirst.com/en/screencasts/-h4afY7FhN/)